### DISCUSSION TITLE
File size of playlist

### DISCUSSION THREAD

**Discussion by andiluechi** [strawberrymusicplayer/strawberry#1394] [Ideas]
How can the file size of a playlist be displayed?

For example, if I have 3 tracks in a playlist and select them, I can see 3 selected of 3 tracks and the maximum playback time at the bottom. Is it also possible to display the maximum file size of this 3 tracks. Or would that be a new feature?

**Comment by jonaski** [comment]
Currently not possible, suggest it on: https://forum.strawberrymusicplayer.org/category/6/feature-suggestions
